.buildkite/launcher_build_steps.yaml

```diff
@@ -0,0 +1,33 @@
+steps:
+  - label: ":linux: :habicat: core/hab-launcher"
+    command: .buildkite/scripts/build_component.sh launcher
+    agents:
+      queue: habitat-release
+    env:
+      BUILD_PKG_TARGET: "x86_64-linux"
+
+  - label: ":linux: :two: :habicat: core/hab-launcher"
+    command: .buildkite/scripts/build_component.sh launcher
+    agents:
+      queue: habitat-release
+    env:
+      BUILD_PKG_TARGET: "x86_64-linux-kernel2"
+
+  - label: ":windows: :habicat: core/hab-launcher"
+    command: .buildkite/scripts/build_component.ps1 -Component launcher
+    agents:
+      queue: windows-docker
+    env:
+      HAB_BLDR_CHANNEL: unstable
+    plugins:
+      - docker#v2.0.0:
+          image: "chefes/windows-base:latest"
+          shell: [ "powershell", "-Command" ]
+          environment:
+            - HAB_BLDR_CHANNEL
+            - HAB_AUTH_TOKEN
+            - BUILDKITE_JOB_ID
+            - BUILDKITE_AGENT_ACCESS_TOKEN
+
+  - wait
```

Comment:
Assuming each build is serialized? We should build the windows-service after the windows launcher.

Reply:
Yup, there's a space immediately after that line where we'll add it :) 

Need to coordinate with @smacfarlane for that, so it'll probably come after this PR.